octobot_services/notifier/abstract_notifier.py

```diff
@@ -147,4 +147,4 @@ async def _create_notification_channel_if_not_existing() -> channels.Channel:
 
     def _is_notification_category_enabled(self, notification):
         return constants.CONFIG_CATEGORY_NOTIFICATION in self.config and \
-               notification.category.value in self.config[constants.CONFIG_CATEGORY_NOTIFICATION]
+               self.config[constants.CONFIG_CATEGORY_NOTIFICATION].get(notification.category.value, True)
```

Comment:
Enabled by default ?

Reply:
For a new category that would not be in config for some reason, I think it makes more sense yes (all current categories are in config and required by config schema)